docs/quick-start-docker.md

```diff
@@ -85,14 +85,13 @@ Add a new data source with with these settings:
 * Name: `metrictank`
 * Default: `yes`
 * Type: `Graphite`
-* Url: `http://localhost:8080`
-* Access: `direct` (not `proxy`)
+* Url: `http://graphite`
+* Access: `proxy`
 
 When you hit save, Grafana should succeed in talking to the data source.
 
 ![Add data source screenshot](https://raw.githubusercontent.com/raintank/metrictank/master/docs/assets/add-datasource-docker.png)
 
-Note: it also works with `proxy` mode but then you have to enter `http://graphite-api:8080` as uri.
```

Comment:
this would be a good place to mention that users can still connect directly to the datasource on localhost:8080.